cmd/update.go

```diff
@@ -109,47 +114,88 @@ func runUpdate(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	logrus.Infof("updating stack %s", stackName)
-
 	stacks, err := stackService.GetStacks(ctx)
 	if err != nil {
 		return err
 	}
+
+	options := stackservice.NewStackManagementOptions(stackName).WithHappyConfig(happyConfig).WithStackService(stackService).WithBackend(backend).WithStackTags(stackTags)
+
 	stack, ok := stacks[stackName]
 	if !ok {
-		return errors.Errorf("stack %s not found", stackName)
+		// Stack does not exist
+		if !force {
+			return errors.Errorf("stack '%s' does not exist, use --force or 'happy create %s' to create it", stackName, stackName)
+		} else { // Force creation of the new stack
```

Comment:
```suggestion
		} 
		// Force creation of the new stack
```
you don't need the if/else here